DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateGroups/EditGroupNameViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using System.ComponentModel;
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class EditGroupNameViewModel
+    {
+        public EditGroupNameViewModel() { }
+
+        public EditGroupNameViewModel(string groupName)
+        {
+            GroupName = groupName;
+        }
+
+        [DisplayName("Group name")]
+        [StringLength(100, ErrorMessage = CommonValidationErrorMessages.StringMaxLengthValidation)]
+        [Required]

```

Comment:
Does this produce the correct error message when the form isn't filled? "Enter a group name"

Reply:
Yes, it does

![image](https://user-images.githubusercontent.com/90783892/143401793-345173b0-c573-4171-a032-cd7cab654076.png)

Reply:
Yeah that's the incorrect message, hence the question. Nothing entered should produce the error message "Enter a group name"

Reply:
Sorry, I have added error message. Please see screenshot below
![image](https://user-images.githubusercontent.com/90783892/143425389-9aba48a2-e0a7-4d2b-b84c-ccdea9a2b0be.png)